tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -370,6 +370,12 @@ def create_parser(product_choices=None):
         help=("Reuse a window across `testharness.js` tests where possible, "
               "which can speed up testing. Also useful for ensuring that the "
               "renderer process has a stable PID for a debugger to attach to."))
+    chrome_group.add_argument(
+        "--quit-gracefully",
```

Comment:
Can you describe the overall procedure to use rr? would like to see if there is another way to ask the browser to quit grafefully.

The document below talks about how to run a wrapper with chrome.
https://docs.google.com/document/d/1teVSVaOAB-iArRYB0kN3XoFiuh68OGcvqSCjY1rfEqE/edit?resourcekey=0-CuV-vrFhfbNcxzVD6rodLg&tab=t.0#heading=h.m58plk1x1smf

Reply:
The end state is for `run_wpt_tests.py --wrapper='rr record'` to work. See the wrapper-side change here: https://crrev.com/c/6271108, and https://crbug.com/339644807#comment12, which the changes implement.

I looked at the document but don't think it accomplishes our goal because neither `--user-data-dir` nor `--log-net-log` are passed, meaning [`chromedriver` will always resort to SIGKILL][0]. This makes ending the session faster for most `chromedriver` clients, but means `rr` won't get a chance to write the recording.

[0]: https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/chromedriver/chrome/chrome_desktop_impl.cc;l=203-206;drc=b5c6f8ac5bb0947e3b772b32350132dfa6426847;bpv=0;bpt=0